portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsImageField.testcase

```diff
@@ -494,16 +494,6 @@ definition {
 	@priority = 4
 	test NotPossibleSubmitLargeImage {
 		task ("Given a Form with an upload field created in a virtual instance") {
-			HeadlessPortalInstanceAPI.addPortalInstance(
-				domain = "www.able.com",
-				portalInstanceId = "www.able.com",
-				virtualHost = "www.able.com");
-
-			User.firstLoginPG(
-				password = "test",
-				userEmailAddress = "test@www.able.com",
-				virtualHostsURL = "http://www.able.com:8080");
-
```

Comment:
Hey @Luizcarlosqueiroz, this test was created to cover https://liferay.atlassian.net/browse/LPS-186215 which is reproducible in a new instance. The other changes look good to me!

Reply:
But considering that the test already creates another instance, is it ok?
I forgot to add this information to the description of the PR.